packages/cli/lib/index.js

```diff
@@ -79,6 +79,7 @@ prog
 	.option('--src', 'Specify source directory', 'src')
 	.option('--cwd', 'A directory to use instead of $PWD', '.')
 	.option('--esm', 'Builds ES-2015 bundles for your code.', true)
+	.option('--no-clear', "Don't clear the console", false)
```

Comment:
Should we rethink if we want to keep this `false`?
// @lukeed @ForsakenHarmony

Reply:
Won't it be too cluttered for users who are just getting started?

Reply:
This doesn't make sense, actually. It should be `('--clear', '...', true)` and then people can disable it thru the `--no-clear` flag

Reply:
@lukeed  Updated.

Reply:
isnt it how most CLI's out there work today?

Reply:
They could, yes, but it makes no sense _for us_ because our display format has been to show the `default: true` version -- looking at `esm` directly above.

Reply:
I don't necessarily agree with `--clear, true` as long as `--no-clear` doesn't get automatically documented by sade 👀

People get confused all the time